libs/coin-modules/coin-near/src/api/node.ts

```diff
@@ -247,9 +285,10 @@ export const getStakingPositions = async (
 
 export const getValidators = makeLRUCache(
   async (): Promise<NearRawValidator[]> => {
+    const currencyConfig = getCoinConfig();
     const { data } = await network({
       method: "POST",
-      url: getEnv("API_NEAR_ARCHIVE_NODE"),
+      url: currencyConfig.infra.API_NEAR_PUBLIC_NODE,
       data: {
         jsonrpc: "2.0",
         id: "id",
```

Comment:
Now using Public endpoint for data unrelated to Near account (no sensitive data)